Urban-Fun.xcworkspace/xcuserdata/mlb.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -116,5 +116,117 @@
             landmarkType = "7">
          </BreakpointContent>
       </BreakpointProxy>
+      <BreakpointProxy
```

Comment:
Not really blocking feedback but normally things like the xcode project and some related settings files such as this don't need to be checked in and you can set it up to ignore such files in the .gitignore file. Might be useful when working on a larger project with others as such files can generate lots of merge conflicts since they store your specific settings.

Reply:
Okay, understood.